src/sql/workbench/contrib/commandLine/electron-browser/commandLine.ts

```diff
@@ -145,14 +149,14 @@ export class CommandLineWorkbenchContribution implements IWorkbenchContribution,
 				this._notificationService.status(localize('runningCommandLabel', "Running command: {0}", commandName), { hideAfter: 2500 });
 			}
 			await this._commandService.executeCommand(commandName, connectedContext);
-		} else if (profile) {
+		} else if (connectedContext) {
```

Comment:
noticed the issue during my testing, the connection failed but it is still trying to open a query editor. Using connectedContext here is better since we can only do the operation if the connection is successfully made.